### PR TITLE
Update sort help message

### LINKED ISSUE #88: Update HelpMessage for Sort Feature
- [x] Make Help message for sort more meaningful #114

### PR DESCRIPTION
**Pull request by whitesnowx**
Closes #88

### CODE REVIEW

File: src/main/java/staffconnect/model/person/comparators/ModuleComparator.java [hunk 1]
```diff
@@ -15,4 +15,9 @@ public class ModuleComparator implements Comparator<Person> {
     public int compare(Person p1, Person p2) {
         return p1.getModule().toString().compareToIgnoreCase(p2.getModule().toString());
     }
+
+    @Override
+    public String toString() {
+        return "Module by alphabetical order";
```
Comment (tsulim) [inline src/main/java/staffconnect/model/person/comparators/ModuleComparator.java]: Perhaps a better word for this is `alphanumeric` as it contains both alphabets and numbers?

File: src/main/java/staffconnect/model/person/comparators/PhoneComparator.java [hunk 2]
```diff
@@ -15,4 +15,9 @@ public class PhoneComparator implements Comparator<Person> {
     public int compare(Person p1, Person p2) {
         return p1.getPhone().toString().compareToIgnoreCase(p2.getPhone().toString());
     }
+
+    @Override
+    public String toString() {
+        return "Phone by alphabetical order";
```
Comment (tsulim) [inline src/main/java/staffconnect/model/person/comparators/PhoneComparator.java]: Should `alphabetical order` be used for `Phone`?
_Might need to update the image if this is edited._

**Review by tsulim — COMMENTED** [review]
Overall, it looks good to me. However, I noticed some terms in the `toString()` that might not be suitable for their respective areas.

File: src/main/java/staffconnect/model/person/comparators/ModuleComparator.java (hunk 1 above)
Comment (whitesnowx) [reply src/main/java/staffconnect/model/person/comparators/ModuleComparator.java]: oo yes thankyou, i was choosing was complating between lexicographical too

File: src/main/java/staffconnect/model/person/comparators/PhoneComparator.java (hunk 2 above)
Comment (whitesnowx) [reply src/main/java/staffconnect/model/person/comparators/PhoneComparator.java]: ~~will ammend!~~ amended

**Review by Pluiexo — APPROVED** [review]
Lgtm, Lexicographical order would be more accurate, but alphanumerical is fine too

File: src/main/java/staffconnect/model/person/comparators/FacultyComparator.java [hunk 3]
```diff
@@ -16,4 +16,9 @@ public class FacultyComparator implements Comparator<Person> {
     public int compare(Person p1, Person p2) {
         return p1.getFaculty().toString().compareToIgnoreCase(p2.getFaculty().toString());
     }
+
+    @Override
+    public String toString() {
+        return "Faculty by alphanumerical order";
```
Comment (tsulim) [inline src/main/java/staffconnect/model/person/comparators/FacultyComparator.java:22]: I don't think `Faculty` accepts numeric.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -158,7 +158,7 @@ Sorts the list of persons based on specified attribute.
 
 Format: `sort [ATTRIBUTE]`
 
-* By default, sorting is done in ascending order.
+* By default, sorting is done in alphabetical order.
```
Comment (tsulim) [inline docs/UserGuide.md:161]: I think you need to update this accordingly.

**Review by tsulim — COMMENTED** [review]
Looks good, only some minor nits. Can be done in another pull request. Thank you👍

**Pull request merged**